content/en/blog/2022/OpenTelemetry-In-Practice-August.md

```diff
@@ -0,0 +1,55 @@
+---
+title: OpenTelemetry in Practice - Kubernetes & the Collector
+linkTitle: OTel in Practice
+date: 2022-08-11
+author: Rynn Mancuso
+draft: true
+# TODO: remove this line once your post is ready to be published
+# TODO: run the following command before submitting
+#   npx prettier --write content/en/blog/2022/OpenTelemetry-In-Practice-August.md
+---
+
+## About the OpenTelemetry in Practice Series
+
+Welcome to the OpenTelemetry in Practice series! This is a new experiment by
+some OpenTelemetry contributors in the End User Working Group.
+
+We’re aiming to:
+
+- address practical problems that commonly stop development teams from
+  succeeding with OpenTelemetry
+- build stronger connections with developers focused on specific languages
+- improve the experience of implementing OpenTelemetry in production
+
+Each OpenTelemetry in Practice session will include a half hour of lightning
+talks and a half hour of open conversation about the topic. We are looking for
+people to join the OpenTelemetry in Practice team and people to give talks at
+future events - so if you’re interested in shaping these conversations, please
+reach out in the
+[#otel-user-research channel](https://cloud-native.slack.com/archives/C01RT3MSWGZ)
+in the [CNCF Slack](https://slack.cncf.io/).
+
+Our first conversation will be on August 23rd, at 10 AM PDT, [on Zoom](/), and
+the topic is Kubernetes & The Collector. Our lightning talks include:
+
+## Observability Pipelines for Kubernetes - Daniel Kim, New Relic
```

Comment:
IMHO, it would be nice to link to presenter pages and/or profiles (e.g. GH), something like this (add a real link):

```suggestion
### Observability Pipelines for Kubernetes

&mdash; by [Daniel Kim](#), New Relic
```